### DISCUSSION TITLE
Change Splash Screen

### DISCUSSION THREAD

**Discussion by curlerhaus** [guysoft/FullPageOS#555] [General]
I've been playing around with this for a couple weeks now and it's great. I'd like to customize the load screen. What is the best way to do that?

**Comment by guysoft** [comment]
Edit the file here:
https://github.com/guysoft/FullPageOS/blob/devel/src/modules/fullpageos/filesystem/boot/splash.png

It will need to be the same size and format

**Comment by curlerhaus** [comment]
Thank you for the information. I haven't created a custom build before. Is that the Docker instructions listed in the README?

**Reply by guysoft** [reply]
Also you might want to change it here too:
https://github.com/guysoft/FullPageOS/issues/559#issuecomment-2096776998

**Reply by guysoft** [reply]
Yes I suggest you look at these:
https://github.com/guysoft/CustomPiOS/wiki/Building-with-Docker

I personally use docker-compose
